app/views/users/_form.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_for(@user, url: defined?(url) ? url : {}) do |f| %>
+  <%= render "shared/error_messages", object: @user %>
+  <%= f.label :name %>
+  <%= f.text_field :name, class: "form-control" %>
+  <%= f.label :email %>
+  <%= f.email_field :email, class: "form-control" %>
+  <%= f.label :password %>
+  <%= f.password_field :password, class: "form-control" %>
+  <%= f.label :password_confirmation %>
+  <%= f.password_field :password_confirmation, class: "form-control" %>
```

Comment:
Có phải em đã .. quên I18n rồi ko :D

Reply:
không ạ, cái này form_for nó tự map vào dịch của activerecord, đoạn này em dịch nè chị 
```
activerecord:
    models:
      user: "User"
    attributes:
      user:
        name: "Name"
        email: "Email"
        password: "Password"
        password_confirmation: "Password Confirmation"
```

Reply:
không ạ trong cái form for dùng cái activereacord thì nó tự map vào luôn á chị